File: Medii de proiectare si programare/java/lab5/ChatNetworking/src/main/java/com/example/objectProtocol/LogInRequest.java
package com.example.objectProtocol;

import com.example.EmployeeAtOffice;

public class LogInRequest implements Request {
    private final EmployeeAtOffice employeeAtOffice;

    public LogInRequest(EmployeeAtOffice employeeAtOffice) {
        this.employeeAtOffice = employeeAtOffice;
    }

    public EmployeeAtOffice getEmployeeAtOffice() {
        return employeeAtOffice;
    }
}

File: an2/Medii de proiectare si programare/C#/tema7/networking/LogInRequest.cs
using System;
using model.domains;

namespace networking
{
    [Serializable]
    public class LogInRequest : Request
    {
        public LogInRequest(EmployeeAtOffice employeeAtOffice)
        {
            _employeeAtOffice = employeeAtOffice;
        }

        public EmployeeAtOffice _employeeAtOffice { get; }
    }
}